src/index.html

```diff
@@ -16,17 +17,24 @@
       href="style.css"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div data-qa="big">
+      <input
+        class="bar big-bar"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div data-qa="small">
```

Comment:
There should be a 20px spacing between the two search bars as per the design specifications. Consider adding CSS to achieve this spacing.